.circleci/config.yml

```
@@ -8,7 +8,7 @@ orbs:
 executors:
   go:
     docker:
-      - image: circleci/golang:1.17
+      - image: cimg/go:1.17
```

```suggestion
      - image: cimg/go:1.18
```

Latest minor now...

https://page.circleci.com/developer/images/image/cimg/go

---

and now 19 😉 

```suggestion
      - image: cimg/go:1.19
```